Modules/_sqlite/connection.c

```diff
@@ -2173,6 +2173,45 @@ pysqlite_connection_create_collation_impl(pysqlite_Connection *self,
     Py_RETURN_NONE;
 }
 
+/*[clinic input]
+_sqlite3.Connection.set_file_control as pysqlite_connection_set_file_control
+
+    op: int
+      a SQLITE_FCNTL_ constant
+    arg: long
+      argument to pass
+    /
+    dbname: str = NULL
```

Comment:
Align this with the `name` keyword-only parameter of `_sqlite3.Connection.blobopen`.

```suggestion
    name: str = "main"
```